packages/ui-truncate-text/src/TruncateText/index.js

```diff
@@ -105,6 +104,8 @@ class TruncateText extends Component {
   constructor(props) {
     super(props)
     this.state = this.initialState
+    // eslint-disable-next-line react/prop-types
```

Comment:
Please add it to propTypes and defaultProps.

Reply:
Added